libqaul/http-api/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "http-api"
```

Comment:
The crate name should probably be more descriptive like `qaul-http` :sweat_smile: